src/utils/getMaxId.ts

```diff
@@ -0,0 +1,5 @@
+import { Todo } from '../types/Todo';
+
+export const getMaxId = (todos: Todo[]) => {
+  return Math.max(...todos.map(todo => todo.id)) + 1;
```

Comment:
Ensure that the `todos` array is not empty before calling `Math.max`, as calling it with an empty array will result in `-Infinity`. Consider providing a default value, such as `0`, to handle the case where `todos` might be empty.